src/server/render.ts

```diff
@@ -261,10 +261,19 @@ export async function render<Data>(
 
   const renderResults: [Plugin, PluginRenderResult][] = [];
 
+  const pluginRenderOptions = {
+    request: opts.request,
+    lang: opts.lang,
+    context: opts.context,
```

Comment:
The outer object is already called `context` and having another property named `context` inside of it feels strange.

```ts
const myPlugin = {
  name: "my-plugin",
  render(ctx) {
    // This feels weird typing context twice
    const config = ctx.context.config
  }
}
```

What do you say about merging `opts.context` directly into this one?

Reply:
I am not sure about bring all with ```...opts.context```, so I am following PageProps, that already exclude render, which one would be duplicated with ```...opts.context```.

What do you think?

Reply:
@marvinhagemeister up